hasjob/templates/layout.html.jinja2

```
@@ -229,5 +240,20 @@
 
 {% block layoutscripts %}
   {%- if header_campaign %}{{ campaign_script() }}{% endif %}
+  <script src='{{ asset_path('manifest') | safe }}' type="text/javascript"></script>
+  <script src='{{ asset_path('vendor') | safe }}' type="text/javascript"></script>
+  <script src='{{ asset_path('app') | safe }}' type="text/javascript"></script>
+  <script type="text/javascript">
+    if ('serviceWorker' in navigator) {
+      navigator.serviceWorker.register('/service-worker.js')
+      .then(function(registration) {
+        console.log('Service Worker registration successful with scope: ',
```

Perhaps better to raise an exception with Sentry?